ansible-st2/roles/rabbitmq/tasks/main.yml

```diff
@@ -1,33 +0,0 @@
-- name: Ensure rabbitmq repo
-  sudo: true
-  apt_repository:
-    repo: deb http://www.rabbitmq.com/debian/ testing main
-
-- name: Ensure rabbitmq repo key
-  sudo: true
-  apt_key:
-    url: http://www.rabbitmq.com/rabbitmq-signing-key-public.asc
-
-- name: Install rabbitmq packages
-  sudo: true
-  apt:
-    name: "{{ item }}"
-    update_cache: true
-  with_items:
-    - rabbitmq-server
-
-- name: Enable rabbitmq_management plugin
-  sudo: true
-  rabbitmq_plugin:
-    names: rabbitmq_management
-  notify:
-    - restart rabbitmq
-
-- meta: flush_handlers
```

Comment:
Docs here: http://docs.ansible.com/ansible/playbooks_intro.html

Appears that it this will ensure that all handlers are flushed here, otherwise it might wait until the end of this section before executing if I understand things correctly.